src/Controller/CaseController.php

```diff
@@ -649,6 +644,13 @@ public function delete(CaseEntity $case, Request $request): Response
     {
         $this->denyAccessUnlessGranted('delete', $case);
 
+        $redirectUrl = $this->generateUrl('default', ['id' => $case->getId()]);
+
```

Comment:
This declaration is very far away from its use. Why is it moved here?

Reply:
Moved back - it was moved as the first solution (rather than throwing an exception) i thought of was redirecting the user.